nixos/modules/virtualisation/amazon-options.nix

```diff
@@ -46,7 +46,7 @@ in {
         defaultText = literalExpression "pkgs.stdenv.hostPlatform.isAarch64";
         internal = true;
         description = ''
-          Whether the EC2 instance is using EFI.
+          Whether the EC2 instance **only** supports EFI.
```

Comment:
Oh I think this variable is actually unused now? I can remove it I think.

Reply:
But maybe with a deprecation warning instead that it's ignored?

Reply:
It's not unused anymore.

Reply:
Yep. I'll resolve